web/yo/app/scripts/services/databaseconnector.js

```diff
@@ -376,16 +376,44 @@ angular.module('oncokbApp')
             }
 
             function updateVUS(hugoSymbol, data, success, fail) {
-                DriveAnnotation
-                    .updateVUS(hugoSymbol, data)
-                    .success(function(data) {
-                        success(data);
-                    })
-                    .error(function(error) {
-                        fail(error);
-                    });
+                if ($rootScope.internal) {
+                    if (dataFromFile) {
+                        success('');
+                    } else {
+                        DriveAnnotation
+                            .updateVUS(hugoSymbol, data)
+                            .success(function(data) {
+                                success(data);
+                            })
+                            .error(function(error) {
+                                var subject = 'VUS update Error for ' + hugoSymbol;
+                                var content = 'The system error returned is ' + JSON.stringify(error);
+                                sendEmail({sendTo: 'dev.oncokb@gmail.com', subject: subject, content: content},
```

Comment:
We have send to dev util.

Reply:
I would like to use the notifyDeveloper function in MainUtils, but that doesn't work because MainUtils is using DatabaseConnector. If I try to use MainUtils in DatabaseConnector, that will create a Circular Dependency error.